docs/architecture/adr-007-light-client-contexts.md

```diff
@@ -214,9 +214,7 @@ trait ClientState<ClientValidationContext, ClientExecutionContext>
 
 The problem with defining all methods directly under `ClientState` is that it would force users to use fully qualified notation to call any method.
 
-This arises from the fact that no method uses both generic parameters. [This playground] provides an explanatory example. Hence, our solution is to have all methods in a trait use every generic parameter of the trait to avoid this problem.
-
-[This playground]: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=da65c22f1532cecc9f92a2b7cb2d1360
```

Comment:
Why remove this playground link?

Reply:
The link is not working anymore and breaks the `md-link-check` job.

Reply:
Weird, the link works for me. But if it's breaking the link check, I don't think it's worth figuring out why; easier to just remove it 👍

Reply:
Gave it another try. Yea, sounds like it's working now. Reverted that! Not sure what was the issue.